crates/autopilot/src/domain/cow_amm.rs

```diff
@@ -0,0 +1,25 @@
+use {primitive_types::H160, std::collections::HashSet};
+
+/// Cow AMM
+/// The list of owners is initialized with the values specified in the autopilot
+/// configuration file, and it is extended with all the CoW AMM owners which are
+/// deployed by the CoW AMM helper contract
+#[derive(Clone)]
+pub struct CowAmm {
```

Comment:
We actually don't need this. But I thought it is nice to have a struct at domain level which takes care of this, potentially extended with more functions in the future (?).